src/Sylius/Bundle/ApiBundle/Validator/Constraints/ZoneCannotContainItselfValidator.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Validator\Constraints;
+
+use Sylius\Component\Addressing\Model\ZoneMemberInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Webmozart\Assert\Assert;
+
+final class ZoneCannotContainItselfValidator extends ConstraintValidator
```

Comment:
Wasn't this covered already? If so - thanks :D

Reply:
I wonder if it couldn't be added in `AddressingBundle`. And do we have a behat scenario for this behaviour? Because if we don't have it I would suggest to add it in a separate PR? However, thanks for paying attention to this 😄

Reply:
If we don't have a Behat scenario for this behaviour, it should be definitely done separately 👌